PHP/PHP.sublime-syntax

```diff
@@ -11,31 +15,93 @@ file_extensions:
   - phps
   - phpt
   - phtml
+
 first_line_match: '^(#!.*[^-]php[0-9]?|<\?php)\b'
-scope: embedding.php
+
 contexts:
-  main:
-    - match: ''
-      push: scope:text.html.basic
-      with_prototype:
-        - match: <\?(?i:php|=)?(?![^?]*\?>)
-          scope: punctuation.section.embedded.begin.php
-          push:
-            - meta_scope: meta.embedded.block.php
-            - meta_content_scope: source.php
-            - include: php-end-tag-pop
-            - include: scope:source.php
-        - match: <\?(?i:php|=)?
-          scope: punctuation.section.embedded.begin.php
-          push:
-            - meta_scope: meta.embedded.line.php
-            - meta_content_scope: source.php
-            - include: php-end-tag-pop
-            - include: scope:source.php
+  prototype:
+    - meta_prepend: true
+    - include: php-tag-start
+
+  tag-attribute-value-content:
```

Comment:
The idea behind `tag-attribute-value-content` and `strings-common-content` is to enable proper string interpolation. Contexts included here should at least contain a `- clear_scopes: 1` in order to clear `string` scopes from embedded/interpolated code. Not doing so might disable any kind of auto-completion features for those code sections.

Reply:
see: https://github.com/FichteForks/Packages/pull/8